pom.xml

```diff
@@ -639,6 +646,10 @@
         <artifactId>org.linkedin.zookeeper-impl</artifactId>
         <version>${linkedin-zookeeper-version}</version>
         <exclusions>
+          <exclusion>
+            <groupId>ant</groupId>
+            <artifactId>ant</artifactId>
+          </exclusion>
```

Comment:
As prior comment, this exclusion doesnt seem like it would be doing anything, as org.linkedin.zookeeper-impl wasnt showing a dependency on "ant:ant" previously, only "org.apache.ant:ant". Seems like it can be removed.